src/organizerproxy.h

```diff
@@ -81,4 +81,60 @@ class OrganizerProxy : public MOBase::IOrganizer
 
 };
 
+

```

Comment:
I would move the 3 dummy interfaces in a separate header (like `dummies.h`). Since these contains no logic these could be header only.

Reply:
There are five dummy implementations, they're contained in the same file as their interface. Do you still think it'd be better if they were all in the same dummy file?

Reply:
From my point of view yes, or maybe in separate dummy files. I don't really like them being in proxy files.

Reply:
Dummies are gone.